ugvc/methylation/concat_methyldackel_csvs.py

```diff
@@ -72,22 +88,18 @@ def run(argv: list[str]):
     try:
         # check if input files exist
 
-        in_file_name = args.input
-        df_csv_output = pd.DataFrame()
-
-        if os.path.isfile(in_file_name):
-
-            with open(in_file_name, "r", encoding="utf-8") as file:
-                csvreader = csv.reader(file)
-                list_csv_files = next(csvreader)
-
-            for csv_file in list_csv_files:
-                metric_name = re.findall(r"MethylDackel([A-Za-z]+)", csv_file, re.IGNORECASE)
-                metric_name = metric_name[-1]
-                df_csv = pd.read_csv(csv_file)
-                df_csv["table"] = metric_name
-                df_csv_output = pd.concat([df_csv_output, df_csv])
-
+        df_mbias = pd.read_csv(args.mbias)
+        df_mbias["table"] = "Mbias"
```

Comment:
```suggestion
        df_csv_output = pd.concat((pd.read_csv(filename).assign(table=table) for table, filename in (("Mbias, args.mbias), ("MbiasNoCpG", args.mbias_non_cpg), ...)))
```
Instead of these 10 lines